spec/controllers/answers_controller_spec.rb

```diff
@@ -14,51 +16,48 @@
         expect { post :create, question_id: question, answer: attributes_for(:answer), format: :js }
                .to change(question.answers, :count).by(1)
       end
-
-      it 'render create template' do
-        post :create, question_id: question, answer: attributes_for(:answer), format: :js
-        expect(response).to render_template 'create'
-      end
     end
 
     context 'with invalid attributes' do
       it 'does not save the answer' do
-        expect { post :create, question_id: question, answer: attributes_for(:invalid_answer), format: :js }
-               .to_not change(Answer, :count)
-      end
-
-      it 'render create template' do
-        post :create, question_id: question, answer: attributes_for(:invalid_answer), format: :js
-        expect(response).to render_template 'create'
+        expect { post :create,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.